frontend/src/components/Jobs/JobPageForSeekers.js

```diff
@@ -434,9 +434,9 @@ function JobPageForSeekers() {
                         <hr/>
                         <Card.Text>{posting.description}</Card.Text>
                         {posting.skills && Array.isArray(posting.skills) && (
-                          <div style={{ display: "flex", flexDirection: "row" }}>
+                          <div className="skill-card" style={{ flexDirection: "row" }}>
                             {posting.skills.map((skill) => (
-                              <span key={skill} className="skills-btn">
+                              <span key={skill} className="skills-btn-seeker">
```

Comment:
I see you made a separate CSS class to further the customization of the skills button.